docs/quick_start.md

```diff
@@ -0,0 +1,220 @@
+# Quick Start
+
+This tutorial (10 to 15 minutes) will help you get started with notion-sdk-py.
+
+## Setup
+
+### Pre requisites
+
+- Make sure you have `python` and `pip` properly installed in your system.
+
+    ```shell
+    python --version
+    pip --version
+    ```
+
+- Create a new directory and move into it to follow along with this tutorial.
+
+    ```shell
+    mkdir learn-notion-sdk-py && cd learn-notion-sdk-py
+    ```
+
+### Installation
+
+- Create a virtual environment and activate it.
+
+    ```shell
+    python -m venv .venv && source .venv/bin/activate
+    ```
+
+- Install `notion-sdk-py` using `pip`
+
+    ```shell
+    pip install --upgrade notion-client
+    ```
+
+### Create integration
+
+- Go to [notion.so/my-integrations](https://www.notion.so/my-integrations)
+to create an integration. Copy the token given by Notion.
+
+- In your working directory, create file named `.env` with
+
+    ```text
+    NOTION_TOKEN=secret_abcd12345
+    # put the real value here
+    ```
+
+### Load secrets
+
+- Install the package [`python-dotenv`](https://pypi.org/project/python-dotenv/)
+which we will use to load our `.env` file.
+
+    ```shell
+    pip install python-dotenv
+    ```
+
+- Create a python module called `settings.py` with the following code.
+
+    ```python
+    # settings.py
+
+    import os
+
+    from dotenv import load_dotenv
+
+    load_dotenv()
+
+    NOTION_TOKEN = os.getenv("NOTION_TOKEN")
+
+    ```
```

Comment:
Why using dotenv here? I think it's much simpler without:

```suggestion
- Make it available in your environment:

    ```shell
    export NOTION_TOKEN=secret_abcd12345`
    ```
```

Reply:
I understand what you are trying to say. `export` is a much simpler way. But it's not persistent. Once you close your terminal, you need to export again. I personally find that annoying. I don't want to change .bashrc for project-level environment variables. So `.env` is a nice way. 

For the sake of simplicity of the tutorial, you may decide to use export. 

It's always a trade-off. No particular way is perfect. The final decision is up to you.

Reply:
Then let's just add a note to say this. :)

> Don't forget that `export` only puts the variable in the environment of the current shell. If you don't want to redo this step for every new shell, add the line in your shell configuration or use a configuration library like [dotenv](https://github.com/theskumar/python-dotenv).

Reply:
that's brilliant. I am putting a commit.

Reply:
![image](https://user-images.githubusercontent.com/66209958/118997643-72671500-b9a6-11eb-8513-3cb27a2fbafd.png)